### DISCUSSION TITLE
swipeaction组件操作问题

### DISCUSSION THREAD

**Discussion by lzzhanglin** [ant-design/ant-design-mobile#5724] [Q&A] [answered]
删除某一项数据的时候，onAction 或者rightActions里面的onClick函数，怎么传递列表数据啊，第二个参数event里面包含了swipeaction的参数key吗
![IMG_20221011_110327](https://user-images.githubusercontent.com/21311138/194988101-7dd8b2c6-670c-4cb7-b7b1-8cd5941fd822.jpg)
![IMG_20221011_110658](https://user-images.githubusercontent.com/21311138/194988122-c600b28e-807a-4a84-beb4-47609ff20575.jpg)

**Comment by lzzhanglin** [comment]
补上onaction函数的图
![Uploading IMG_20221011_110525.jpg…]()

**Comment by miracles1919** [comment, marked as answer]
目前不支持，感觉可以加一个标识符，在事件操作中能获取到 @zombieJ

**Reply by lzzhanglin** [reply]
多谢回复
